docs/application/dotnet/guides/nui/nui-components/Button.md

```diff
@@ -136,18 +136,18 @@ Following output is generated when the Button is created using the defined style
 
 ![ButtonProperty](./media/ButtonProperty.PNG)
 
-## Responding to ClickEvent
+## Responding to Clicked
 
 When you click a Button, the Button instance receives a click event.
 You can declare the click event handler as follows:

```

Comment:
Suggestion. How about changing "click" to "clicked"?

```suggestion
When you click a Button, the Button instance receives a clicked event.
You can declare the clicked event handler as follows:
```

Reply:
Thank you for your suggestion.
I have updated the patch as you suggested.